### ISSUE TITLE
Ember dayjs webpack compatibility

### ISSUE THREAD

**Issue by Shishouille** [sinankeskin/ember-dayjs#288]
Hi ! First things first thanks for creating this addon :)

I tried to install it but I think there is an issue with the webpack config. I can't load the locales because it doesn't find it. I have this message error on a the start of the app : 
![image](https://user-images.githubusercontent.com/55913724/178279645-f3b05250-b16b-40a5-bb12-3046baeceaed.png)

Here's my `ember-cli-build` file: 
```
// ember-cli-build.js
const EmberApp = require('ember-cli/lib/broccoli/ember-app');
const nodeSass = require('node-sass');
const { ESBuildMinifyPlugin } = require('esbuild-loader');
const { Webpack } = require('@embroider/webpack');
const currentYear = new Date().getFullYear();

const isProduction = EmberApp.env() === 'production';
const isTest = EmberApp.env() === 'test';

module.exports = function (defaults) {
  let app = new EmberApp(defaults, {
     ...
    'ember-dayjs': {
      locales: ['en-gb', 'fr', 'de', 'es'],
      plugins: ['utc', 'localizedFormat']
    },

    autoImport: {
      forbidEval: true,
      webpack: {
        plugins: [],
        devtool: 'source-map',
        optimization: {
          minimizer: [
            new ESBuildMinifyPlugin({
              legalComments: 'none',
              sourcemap: !isProduction,
              minify: isProduction,
              css: true
            })
          ]
        }
      }
    },

    // Autoprefixer options
    autoprefixer: {
      enabled: true,
      cascade: true,
      sourcemap: !isProduction
    },

    tests: isTest,
    hinting: isTest
  });

  return require('@embroider/compat').compatBuild(app, Webpack, {
    staticAddonTestSupportTrees: true,
    staticAddonTrees: true,
    staticHelpers: true,
    staticModifiers: true,
    staticComponents: true,
    splitAtRoutes: [
      'login',
    ], 
    packagerOptions: {
      webpackConfig: {
        devtool: 'source-map',
        node: {
          global: true
        }
      }
    },
  });
};
```

Do you have any ideas on how to fix or update this ? It would be nice maybe to pass the ember-cli-build config and set the dayjs plugin the same way you set the locale. 

What do you think ? Has someone been already confronted to this issue ? 
Thanks!

**Comment by sinankeskin** [comment]
Oh looks like you are using embroider to build. Embroider does something like this (because I'm using eager load with app.import) but If I change to dynamic import both locale and plugins are probably will work fine with embroider.

Also I like the idea of lazy loading. I'll update and let you know @Shishouille 

Unnecessary info but `Şirin` means cute in Turkish. Thank you for the feedbacks.

**Comment by Shishouille** [comment]
That's sweet, thanks @sinankeskin ! If you need any help tell me.

And thanks for the info haha 😊 . It has kind of the same meaning in farsi.

**Comment by sinankeskin** [comment]
Hey turns out that why my mistake that locale full name needs to have underscore instead of hyphen. I've released v0.8.1 and tested with brand new embroider app and works ok now. 

Could you please test with your app as well? Thanks @Shishouille

**Comment by Shishouille** [comment]
Yes it works on mine too ! Thanks a lot @sinankeskin, you were fast on this one haha. 
I'll close the issue :)
